pkg/replication/replicator.go

```diff
@@ -108,20 +108,22 @@ func (r replicator) InitializeReplication(
 		ignoreControllers,
 		concurrentRealityRequests,
 		true,
+		false,
 	)
 }
 
 func (r replicator) InitializeReplicationWithCheck(
```

Comment:
if daemon_sets are the only caller (and they only call it once, with mostly fixed args), why not just say `InitializeDaemonSetReplication` and remove most of the args? The only arg that needs to remain in this call is `concurrentRealityRequests`.